test/cmdlineTests/~soljson_via_fuzzer/test.sh

```diff
@@ -10,7 +10,7 @@ cd "$SOLTMPDIR"
 "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/test/
 "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/
 
-echo ./*.sol | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --quiet --input-files
-echo ./*.sol | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --without-optimizer --quiet --input-files
+grep -L --include="*.sol" "pragma experimental solidity;" * | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --quiet --input-files
```

Comment:
To reduce the chance of false-positives:

```suggestion
grep -L --include="*.sol" "^pragma experimental solidity;$" * | xargs -P 4 -n 50 "${SOLIDITY_BUILD_DIR}/test/tools/solfuzzer" --quiet --input-files
```

Better skip too few than too many.